code/modules/admin/view_variables/reference_tracking.dm

```diff
@@ -70,52 +71,81 @@
 
 	qdel_and_find_ref_if_fail(src, TRUE)
 
+#ifdef REFERENCE_TRACKING_DEBUG
+///Stores info about where refs are found, used for sanity checks
+/datum/var/list/found_refs
```

Comment:
Don't do this--it messes with SpacemanDMM (you likely remember this from LINDA tiles).

Just do this in wherever `/datum` is:

```
#ifdef REFERENCE_TRACKING_DEBUG
    /// bla bla bla
    var/list/found_refs
#endif
```

Reply:
Just in case you were wondering, no SpacemanDMM won't pick this up because it's behind an `ifdef`, but we have a history of applying `|| defined(SPACEMANDMM)` later on.

Reply:
if you apply spacemandmm to ref tracking, the linter's gonna fail. but sure, I'll put this in /datum